integration-tests/parachain-runtime-mock/src/lib.rs

```diff
@@ -228,6 +229,10 @@ impl cumulus_pallet_xcmp_queue::Config for ParachainRuntime {
 	type ControllerOriginConverter = XcmOriginToCallOrigin;
 	type PriceForSiblingDelivery = ();
 	type WeightInfo = ();
+	type ExecuteDeferredOrigin = EnsureRoot<AccountId>;
+	type MaxDeferredMessages = ConstU32<100>;
```

Comment:
what happens to the messages if this buffer is filled?

Reply:
We drop the message.
https://github.com/galacticcouncil/cumulus/pull/3/files#diff-3d3febd8cfcd42ba5bb949d63fdafb9a71da3db1feb35a45601404b64fcd70c4R841